### PR TITLE
made review form same style as submit sighting form. Made disabled in…

### PR DESCRIPTION
**Pull request by emilypatsko**
…puts clearer

### CODE REVIEW

File: ClientApp/src/Api/apiClient.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
-﻿import authService from '../components/api-authorization/AuthorizeService'
+﻿import { ExecOptionsWithStringEncoding } from 'node:child_process';
```
Comment (ChristianMcc) [inline ClientApp/src/Api/apiClient.ts]: What's this?

**Review by ChristianMcc — COMMENTED** [review]
Looks good to me apart from an odd import!

File: ClientApp/src/Api/apiClient.ts (hunk 1 above)
Comment (emilypatsko) [reply ClientApp/src/Api/apiClient.ts]: i really don't know where that's come from! I definitely didn't type it....

Comment (emilypatsko) [reply ClientApp/src/Api/apiClient.ts]: it must have put it there automatically from a typo. fixed!

**Review by ChristianMcc — APPROVED** [review]

**Pull request merged**